webapp/channels/src/components/create_user_groups_modal/create_user_groups_modal.tsx

```diff
@@ -187,10 +184,7 @@ export default class CreateUserGroupsModal extends React.PureComponent<Props, St
                                         this.goBack();
```

Comment:
thanks for removing it from the icon but we still need the aria label of this button to be in formatMessage

Reply:
Ok